third_party/terraform/utils/compute_shared_operation.go

```diff
@@ -1,25 +1,16 @@
 package google
 
-import (
-	computeBeta "google.golang.org/api/compute/v0.beta"
-	"google.golang.org/api/compute/v1"
-)
-
-func computeSharedOperationWait(client *compute.Service, op interface{}, project string, activity string) error {
-	return computeSharedOperationWaitTime(client, op, project, 4, activity)
+func computeSharedOperationWait(config *Config, op interface{}, project, activity string) error {
+	return computeSharedOperationWaitTime(config, op, project, activity, 4)
 }
 
-func computeSharedOperationWaitTime(client *compute.Service, op interface{}, project string, minutes int, activity string) error {
+// This is a shell around computeOperationWaitTime. It was originall meant to type switch between Beta and GA wait
+// operations but it now serves to differentiate handwritten resource calls to computeWait from generated. This method
+// should be eventually removed when the distinction is no longer needed.
+func computeSharedOperationWaitTime(config *Config, op interface{}, project, activity string, minutes int) error {
```

Comment:
Since this does nothing anymore, want to remove it and call `computeOperationWaitTime` directly?

Reply:
Yeah it was on my mind too (hence the checkbox above) I'll pull it post 3.0

Reply:
done.